VoiceAttackResponder/VoiceAttackPlugin.cs

```diff
@@ -1265,6 +1265,7 @@ private static void setCommanderValues(Commander cmdr, ref dynamic vaProxy)
                 vaProxy.SetText("Debt (spoken)", Translations.Humanize(cmdr?.debt));
 
                 vaProxy.SetText("Title", cmdr?.title);
+                vaProxy.SetText("Gender", cmdr?.gender);
```

Comment:
Would suggest `cmdr?.gender ?? "Neither"` rather than leaving it as null if the user hasn't set one.

Reply:
An enum might be better than a string but I'm not familiar with VA's conventions in that regard.

Reply:
I'm not sure it can be unset.  The default is "Male" as it's the first radio button on the Commander Details tab.  Perhaps if the UI is never started after an install, it could be, so that's worth testing out.

There is a race condition for profile writers in VA where if you query the value prior to the plugin initializing, you'll get a "Not Set", but that is expected and has to be handled at the Voice Attack end.

Regarding the enum, that doesn't really follow the current pattern.  Environment, for example, also has three values.